remotespark/RemoteSparkMagics.py

```diff
@@ -161,14 +160,12 @@ def _get_client_keys(self):
         return "Possible endpoints are: {}".format(self.client_manager.get_endpoints_list())
 
     def _send_command(self, client, command, language):
-        if language == Constants.lang_scala:
-            return client.execute_scala(command)
-        elif language == Constants.lang_pyspark:
-            return client.execute_pyspark(command)
-        elif language == Constants.lang_sql:
-            return client.execute_scala_sql(command)
-        elif language == Constants.lang_pysql:
-            return client.execute_pyspark_sql(command)
+        if language == "sql":
```

Comment:
The languages from constants are for the session. Now, when doing code submission, you are using a session that is tied to a language ("default") or the sqlcontext in that language ("sql"). 

You won't get a language not supported exception because that verification is done at session creation time, not code submission.

Did I explain myself?